snippets/snippets-ts.json

```diff
@@ -1 +1,250 @@
-{}
\ No newline at end of file
+{
+	"fastify hello server": {
+		"scope": "typescript",
+		"prefix": "ffhello",
+		"body": [
+			"import fastify from 'fastify'",
+			"const app = fastify({ logger: true })",
+			"app.get('/', async (request, reply) => {",
+			"  return { hello: 'world' }",
+			"})",
+			"app.post('/', async (request, reply) => {",
+			"  return request.body",
+			"})",
+			"app.listen({ port: 8080 })"
+		],
+		"description": "Create an hello world server"
+	},
+	"fastify register": {
+		"scope": "javascript,typescript",
```

Comment:
We should fix all the `scope`s in the file

Reply:
Yes